x-pack/plugins/endpoint/common/types.ts

```diff
@@ -55,22 +57,22 @@ export interface AlertResultList {
   /**
    * The index of the requested page, starting at 0.
    */
-  request_page_index: Maybe<number>;
+  request_page_index?: number;
 
   /**
    * The offset of the requested page, starting at 0.
    */
-  result_from_index: Maybe<number>;
+  result_from_index?: number;
 
   /**
    * A cursor-based URL for the next page.
    */
-  next: Maybe<string>;
+  next?: string;
```

Comment:
@peluja1012 So I thiiiink this means that `next` and `prev` won't appear in the response at all if there's no next/prev page... is that intended? I was trying to ensure that they're always returned, even if they are undefined/null